src/sync/impl/sync_metadata.cpp

```diff
@@ -77,6 +78,7 @@ realm::Schema make_schema()
             {c_sync_provider_type, PropertyType::String}
         }},
         {c_sync_profile, {
+            {c_sync_profile_name, PropertyType::String|PropertyType::Nullable},
```

Comment:
[q] What is the purpose of this field?

Reply:
I don't know. The property was something you added to `SyncUserProfile` with the comment: "The full name of the user." Since it was the only property not persisted, I assumed that was an oversight and added it in for consistency. Should it instead be removed from `SyncUserProfile`?